runtime/lua/vim/lsp/handlers.lua

```diff
@@ -565,6 +565,47 @@ M[ms.callHierarchy_incomingCalls] = make_call_hierarchy_handler('from')
 --- @see # https://microsoft.github.io/language-server-protocol/specifications/specification-current/#callHierarchy_outgoingCalls
 M[ms.callHierarchy_outgoingCalls] = make_call_hierarchy_handler('to')
 
+--- @private
```

Comment:
No need to mark local functions as `private` (they are already scoped to the module).